src/DynamicExpresso.Core/InterpreterOptions.cs

```diff
@@ -25,6 +25,10 @@ public enum InterpreterOptions
 		/// <summary>
 		/// Load all default configurations: PrimitiveTypes + SystemKeywords + CommonTypes
 		/// </summary>
+		LateBindObject = 16,
+		/// <summary>
+		/// Allow treating expressions of type Object as dynamic
+		/// </summary>
```

Comment:
I think that you should swap comments. The documentation should be on top ...

Reply:
Oh, my bad. Already done. Thanks!